contrib/src/main/java/com/datatorrent/contrib/dimensions/AbstractAppDataDimensionStoreHDHT.java

```diff
@@ -127,7 +127,7 @@
   private Unifier<String> queryResultUnifier;
 
   protected long responseDelayMillis;
-  
+
```

Comment:
All format/codestyle related changes must be done in a separate commit (or even in a separate PR with separate JIRA) using Malhar Jenkins user.